### ISSUE TITLE
[Question]: Is there a method to scroll?

### ISSUE THREAD

**Issue by saona-raimundo** [jonhoo/fantoccini#137]
Hi! Thank you for the crate! It has been quite useful for me.

I could not find a method called "scroll" and I checked the `webdriver` crate without luck, so I wanted to ask you directly;
Is there a way to control the website and scroll? 

## Use case

Clicking into an element implies first scrolling into the element into view, and then clicking (by W3 protocol). I wanted to check if an element can be scrolled into view before trying to click on it. 

In my case, being able to scroll an element into view tells me that the website is in some state. Therefore, I can do some operations before clicking this element. 

In short, I want to do something in between scrolling and clicking an element. Is there a way to do it?

**Comment by jonhoo** [comment]
You should be able to use [`Client::execute`](https://docs.rs/fantoccini/0.17/fantoccini/struct.Client.html#method.execute) along with something like [`window.scroll`](https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll) in JavaScript to do this :)

**Comment by JohnScience** [comment]
How do I do this for a `fantoccini::elements::Element`? I work with a list of those.

**Comment by jonhoo** [comment]
@JohnScience I believe you should be able to pass the `Element` to `Client::execute`, and then use the element's `.offsetY` to pass to `scrollTo` entirely in JavaScript

**Comment by JohnScience** [comment]
> @JohnScience I believe you should be able to pass the `Element` to `Client::execute`, and then use the element's `.offsetY` to pass to `scrollTo` entirely in JavaScript

I'm not quite sure how to pass the `Element` of `Client::execute` because of its signature:

```
pub async fn execute(
        &self,
        script: &str,
        mut args: Vec<Json>,
    ) -> Result<Json, error::CmdError> {
      // ...
    }
```

Is there a way to convert `Element` to `Json` that would represent an `Element` or did I get your suggestion wrong?

**Comment by jonhoo** [comment]
@JohnScience You should be able to exactly just serialize it and pass it as one of the args, and then the browser's implementation of WebDriver should ensure it pops out the other end as a reference to an element (see https://www.w3.org/TR/webdriver1/#elements).
